packages/babel-code-frame/test/index.js

```diff
@@ -1,10 +1,30 @@
-import chalk from "chalk";
 import stripAnsi from "strip-ansi";
-
+import { getChalk } from "@babel/highlight";
```

Comment:
The `getChalk` private API is also used in `babel-code-frame` test here so that we can manipulate the chalk instance required by `babel-highlight`. If we remove the `getChalk` API, we may have to fallback to resolving it from babel-highlight.